mesonbuild/modules/i18n.py

```diff
@@ -398,6 +573,23 @@ def itstool_join(self, state: 'ModuleState', args: T.List['TYPE_var'], kwargs: '
 
         return ModuleReturnValue(ct, [ct])
 
+    @FeatureNew('i18n.find_xgettext', '1.7.0')
+    @noPosargs
+    @typed_kwargs(
+        'i18n.find_xgettext',
+        _ARGS,
+        KwargInfo('required', bool, default=True),
```

Comment:
Why not use the shared option that also handles UserFeatureOptions? Or add the disabler argument. There may be something I'm missing here.

Reply:
no longer relevant, I think.